docs/mobile/events.md

```diff
@@ -8,12 +8,27 @@ events on mobile platforms. All mobile events MUST use a namespace of
 
 <!-- toc -->
 
+- [Application unresponsive UI](#application-unresponsive-ui)
 - [Lifecycle instrumentation](#lifecycle-instrumentation)
   * [iOS](#ios)
   * [Android](#android)
 
 <!-- tocstop -->
 
+## Application unresponsive UI
+
+This event denotes when an application's UI "freezes", making users unable to interact with the app. It happens when the
+main thread (which is in charge of rendering the UI) gets blocked by a long-running task for a certain amount of time.
+
+The amount of time the UI must be frozen to trigger this event is implementation-specific, however, for Android applications
```

Comment:
Should this be a recommended attribute as well?

Reply:
So far I couldn't think of a use case that would require displaying the amount of time that took to trigger a UI freeze event as part of its attrs. This is because the use cases I currently have in mind would either be a constant, such as in Android, or would be provided by the user in a config param, and in that case I guess it should be implicit to them since they've set it up.

I'm not opposed to adding it though, is just that I would like to have an idea of a valid use case for it to back it up when questions such as payload size come up regarding it.